### ISSUE TITLE
Idea: Consider switching to another backup tool

### ISSUE THREAD

**Issue by gw0** [tozd/docker-rdiff-backup#3]
In my opinion there are better backup tools than `rdiff-backup` with nice features, such as encryption and incremental backups. So maybe it would make sense to consider switching to those. To the best of my knowledge currently the best are:

- https://restic.net/
- https://www.borgbackup.org/

**Comment by mitar** [comment]
incremental backups are supported by rdiff-backup?

But yes, I do not really mind what is used, this is really for me just to backup whole `/srv` to another server. I trust both servers. So encryption is not a critical component here. Deduplication and compress probably.

**Comment by mitar** [comment]
Both of those programs are in Ubuntu, which is good.

**Comment by gw0** [comment]
Yes, both are nice. I am using Borgbackup and Tadej is using Restic. Their are only small differences between them and I would say it basically comes to: do you need to backup to cloud providers and consume more bandwidth (Restic), or do you have SSH access and can install a server-side service (Borgbackup).

**Comment by mitar** [comment]
And how they compare with rdiff-backup?

**Comment by gw0** [comment]
As you said, primarily encryption and deduplication, secondarily compression (Borgbackup) and no need to install anything (Restic).

I found a speed comparison table (if I am not mistaken `duplicity` shares code with `rdiff-backup`):

- https://github.com/gilbertchen/benchmarking
